.pipelines/singletenancy/aks-swift/e2e-step-template.yaml

```diff
@@ -31,25 +30,17 @@ steps:
       addSpnToEnvironment: true
       inlineScript: |
         set -e
-        mkdir -p ~/.kube/
-        echo "Create AKS cluster"
-        make -C ./hack/aks azcfg AZCLI=az REGION=$(REGION_AKS_CLUSTER_TEST)
-        make -C ./hack/aks byocni-up AZCLI=az REGION=$(REGION_AKS_CLUSTER_TEST) SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-${{ parameters.osSku }}-$(make revision) OSSKU=${{ parameters.osSku }}
```

Comment:
This might be a typo in the original code for variable: `OS_SKU` . Is this needed ? by default it will be `Linux`(which satisfies our pipeline but was this added to address the future windows e2e stages for swift ?

Reply:
Referring to documentation, the only values that can be accepted are for az aks nodepool:`AzureLinux, CBLMariner, Mariner, Ubuntu, Windows2019, Windows2022` for az aks create:`AzureLinux, CBLMariner, Mariner, Ubuntu` Considering we are only supporting Ubuntu and Windows2022 I don't see the need to keep it or use it as we already have flow setup in case of Windows2022 being passed.